app/component/AppBarLarge.js

```diff
@@ -25,12 +29,15 @@ const AppBarLarge = ({ titleClicked }, { router, location, config, intl }) => {
     <div>
       <div className="top-bar bp-large flex-horizontal">
         <button className="noborder" onClick={titleClicked}>
-          {config.textLogo ? (
+          {isBrowser && !config.textLogo && logo ? (
```

Comment:
Isn't this wrong, as it makes SSR render the text logo, rather than just having an empty background image?